propolis/src/exits.rs

```diff
@@ -67,6 +76,11 @@ impl From<&vm_exit> for VmExitKind {
             Ok(c) => c,
         };
         match code {
+            vm_exitcode::VM_EXITCODE_VMX => {
+                // TODO: Populate and use this exit type.
+                let _vmx = unsafe { &exit.u.vmx };
+                VmExitKind::Vmx
+            }
```

Comment:
Make this separate from the rustdoc change?

Reply:
Done